docs/install/prerequisites-data.rst

```diff
@@ -0,0 +1,42 @@
+===========================
+Prerequisites for your data
```

Comment:
Pour moi, ces données sont les données initiales minimales (Minimal initial data), donc à préciser dans le titre ou dans un petit texte, surtout maintenant que c'est déplacé plus bas dans la documentation.

Reply:
C'est fait, il y a désormais un message en base de la page [Installation](https://geotrek.readthedocs.io/en/mfu-improve-doc-gta/install/installation.html)  qui renvoie vers la page [Prerequisites for your data](https://geotrek.readthedocs.io/en/mfu-improve-doc-gta/install/prerequisites-data.html) sur laquelle on trouve aussi un message qui précise que les données listées sont celles des données initiales.

Reply:
OK, mais en fait je crois que je comprends pas ce titre de rubrique.
"Importer des données / Pré-requis pour vos données".
En fait le titre devrait uniquement être "Minimal initial data" plutôt, non ?

Reply:
Oui, je reconnais que le titre pouvait porter à confusion, je l'ai changé : https://geotrek.readthedocs.io/en/mfu-improve-doc-gta/install/minimal-initial-data.html